site/content/docs/0.0/getting-started/javascript.md

```diff
@@ -8,4 +8,312 @@ aliases:
+
+  Array.from(document.querySelectorAll('.toast'))
+    .forEach(toastNode => new Toast(toastNode))
+</script>
+```
+
+Compared to JS bundlers, using ESM in the browser requires you to use the full path and filename instead of the module name. [Read more about JS modules in the browser.](https://v8.dev/features/modules#specifiers) That's why we use `'ouds-web.esm.min.js'` instead of `'ouds-web'` above. However, this is further complicated by our Popper dependency, which imports Popper into our JavaScript like so:
+
+<!-- eslint-skip -->
+```js
+import * as Popper from "@popperjs/core"
+```
+
+If you try this as-is, you'll see an error in the console like the following:
+
+```text
+Uncaught TypeError: Failed to resolve module specifier "@popperjs/core". Relative references must start with either "/", "./", or "../".
+```
+
+To fix this, you can use an `importmap` to resolve the arbitrary module names to complete paths. If your [targeted browsers](https://caniuse.com/?search=importmap) do not support `importmap`, you'll need to use the [es-module-shims](https://github.com/guybedford/es-module-shims) project. Here's how it works for OUDS Web and Popper:
+
+<!-- eslint-skip -->
+```html
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="{{< param "cdn.css" >}}" rel="stylesheet" integrity="{{< param "cdn.css_hash" >}}" crossorigin="anonymous">
+    <title>Hello, modularity!</title>
+  </head>
+  <body>
+    <h1>Hello, modularity!</h1>
+    <button id="popoverButton" type="button" class="btn btn-primary btn-lg" data-bs-toggle="popover" title="ESM in Browser" data-bs-content="Bang!">Custom popover</button>
+
+    <script async src="https://cdn.jsdelivr.net/npm/es-module-shims@1/dist/es-module-shims.min.js" crossorigin="anonymous"></script>
+    <script type="importmap">
+    {
+      "imports": {
+        "@popperjs/core": "{{< param "cdn.popper_esm" >}}",
+        "ouds-web": "https://cdn.jsdelivr.net/npm/@ouds/web@{{< param "current_version" >}}/dist/js/ouds-web.esm.min.js"
+      }
+    }
+    </script>
+    <script type="module">
+      import * as oudsWeb from 'ouds-web'
+
+      new oudsWeb.Popover(document.getElementById('popoverButton'))
+    </script>
+  </body>
+</html>
+```
+
+## Dependencies
+
+Some plugins and CSS components depend on other plugins. If you include plugins individually, make sure to check for these dependencies in the docs.
+
+Our dropdowns, popovers, and tooltips also depend on [Popper](https://popper.js.org/docs/v2/).
+
+## Data attributes
+
+Nearly all OUDS Web plugins can be enabled and configured through HTML alone with data attributes (our preferred way of using JavaScript functionality). Be sure to **only use one set of data attributes on a single element** (e.g., you cannot trigger a tooltip and modal from the same button.)
+
+{{< markdown >}}
+{{< partial "js-data-attributes.md" >}}
```

Comment:
Will need to change before we merge it since `OUDS Web 5.2.0` is displayed.

Reply:
Good catch, it's fixed by https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2648/commits/f110360cd348ca319185f103088619689f69b5e2.
I've removed the "As of OUDS Web 5.2.0, " part as it's not necessary to have this precise information for OUDS Web, since OUDS Web starts from v5.3.x.